tests/Neo.Plugins.RpcServer.Tests/UT_RpcServer.Blockchain.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// UT_RpcServer.Blockchain.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Neo.Ledger;
+using Neo.Persistence;
+using System;
+using System.Text;
+
+namespace Neo.Plugins.RpcServer.Tests
+{
+    public partial class UT_RpcServer
```

Comment:
Remove it?

Reply:
please dont, its a part of my ut plan, showing how i will work on them. this pr is just the first pr of many others. please keep it if it wont cause problem.